src/v2/view-builder/components/AuthenticatorEnrollOptions.js

```diff
@@ -42,7 +44,35 @@ const AuthenticatorRow = View.extend({
   },
   minimize: function() {
     this.$el.addClass('authenticator-row-min');
-  }
+  },
+  getTemplateData() {
+    let authenticatorUsageText;
+    const allowedFor = this.model.get('relatesTo').allowedFor;
+
+    if (allowedFor) {
+      switch (allowedFor) {
+      case 'any':
+        authenticatorUsageText = loc('oie.enroll.authenticator.usage.text.access.recovery', 'login');
+        break;
+      case 'recovery':
+        authenticatorUsageText = loc('oie.enroll.authenticator.usage.text.recovery', 'login');
+        break;
+      case 'sso':
+        authenticatorUsageText = loc('oie.enroll.authenticator.usage.text.access', 'login');
+        break;
+      default:
```

Comment:
do we need a default if there are no default action to be performed?

Reply:
I don't think we need it. Removed.